docs/build.md

```diff
@@ -0,0 +1,13 @@
+## Building images related to Kubernetes and Kind.
+
+#### Building Kind-node images:
+Kind node images are built with the version of kubernetes, it can be done by triggering the workflow
+under the `Docker` workflow. As an input parameter, one needs to set the version of `Kubernetes` before
+proceeding to build the node image. The default version to build is `master`
+
+![img.png](img.png)
+
+#### Building base and kind-dependent images.
+These images are built with a new release, triggering this workflow leads to building on images related to storage
+networking, etc. which are internally used by the kind binary. Once the images are built successfully, the corresponding
+versions are updated in the `build-ppc64le.patch` file.
```

Comment:
isn't it required to mention that this .patch file will be used later while installing?

Reply:
Base images are mostly built on every release and pushed to image registry.. It shouldn't be linked in my opinion.